standard/statements.md

```diff
@@ -433,14 +431,14 @@ An *explicity_typed_local_variable_declaration* introduces one or more local var
 
 If a *local_variable_initializer* is present then its type shall be appropriate according to the rules of simple assignment ([§12.21.2](expressions.md#12212-simple-assignment)) or array initialization ([§17.7](arrays.md#177-array-initializers)) and its value is assigned as the initial value of the variable.
 
-#### 13.6.2.4 Ref local variable declarations
+#### 13.6.2.4 Explicitly typed ref local variable declarations
 
 ```ANTLR
-ref_local_variable_declaration
-    : ref_kind type ref_local_variable_declarators
+explicitly_typed_ref_local_variable_declaration
+    : ref_kind type explicitly_typed_ref_local_variable_declarators
     ;
 
-ref_local_variable_declarators
+explicitly_typed_ref_local_variable_declarators
```

Comment:
```suggestion
ref_local_variable_declarators
```

This rule is a list of *ref_local_variable_declarator* so should be named at it, there is no explicit type in the declarator. Note *implicitly_typed_local_variable_declaration* also uses *ref_local_variable_declarator*.